.github/guides/MAPS_AND_AWAY_MISSIONS.md

```diff
@@ -1,43 +1,53 @@
+
+If you are hosting a server, and want randomly picked maps to be played each round, you can enable map rotation in `config/config.txt` and then set the maps to be picked in the `config/maps.txt` file.
 
-All maps have their own code file that is in the base of the _maps directory. Maps are loaded dynamically when the game starts. Follow this guideline when adding your own map, to your fork, for easy compatibility.
+## EDITING MAPS
 
-The map that will be loaded for the upcoming round is determined by reading data/next_map.json, which is a copy of the json files found in the _maps tree. If this file does not exist, the default map from config/maps.txt will be loaded. Failing that, BoxStation will be loaded. If you want to set a specific map to load next round you can use the Change Map verb in game before restarting the server or copy a json from _maps to data/next_map.json before starting the server. Also, for debugging purposes, ticking a corresponding map's code file in Dream Maker will force that map to load every round.
+<b>It is absolutely inadvisable to <i>ever</i> use the mapping utility offered by Dream Maker</b>. It is clunky and dated software that will steal your time, patience, and creative desires.
 
-If you are hosting a server, and want randomly picked maps to be played each round, you can enable map rotation in [config.txt](config/config.txt) and then set the maps to be picked in the [maps.txt](config/maps.txt) file.
+Instead, /tg/station map maintainers will always recommend using one of two modern and actively maintained programs.
+* [StrongDMM](https://github.com/SpaiR/StrongDMM) (Windows/Linux/MacOS)
+* [FastDMM2](https://github.com/monster860/FastDMM2) (Web-based Utility)
 
-Anytime you want to make changes to a map it's imperative you use the [Map Merging tools](https://tgstation13.org/wiki/Map_Merger)
+Both of the above programs have native TGM support, which is mandatory for all maps being submitted to this repository. Anytime you want to make changes to a map, it is imperative you use the [Map Merging tools](https://tgstation13.org/wiki/Map_Merger). When you clone your repository onto your machine for mapping, it's always a great idea to run `tools/hooks/Install.bat` at the very start of your mapping endeavors, as this will install Git hooks that help you automatically resolve any merge conflicts that come up while mapping.
```

Comment:
> Anytime you want to make changes to a map, it is imperative you use the [Map Merging tools](https://tgstation13.org/wiki/Map_Merger)

this seems wrong i have never used map merger, its functions are included in sdmm no?

Reply:
sDMM has a map merger which supports the TGM file format and it can accommodate to TGM map keys... but if you have a merge conflict on a regularly touched, sDMM probably won't be of much help. It's much easier to rebase your branch off master when you have a merge conflict and let mapmerge2 Git hooks do the heavy lifting as opposed to having to remap the section by yourself, and letting it flag any problematic tiles.

It was also important to use mapmerge2 to get the DMM to TGM format conversion done, but if you edit maps in the native TGM format with either FastDMM or StrongDMM, that's already included... it's just that it's better to have them used rather than not used in my personal experience as well as my experience teaching others how to map and resolve merge conflicts as they arise.

Reply:
imperative clearly means you have to do it, when you dont have to do it most of the time

Reply:
Imperative is a hold-over from oranges' initial version of this document- not particularly interested in changing it.